docs/Makefile.am

```diff
@@ -0,0 +1,96 @@
+#
+# Copyright (c) 2022 Cisco Systems, Inc.  All rights reserved.
+#
+# $COPYRIGHT$
+#
+# Additional copyrights may follow
+#
+# $HEADER$
+#
+
+# Random notes:
+#
+# - It would have been nice to auto-generate some of the man page
+#   listing files (e.g., man-openmpi/man3/index.rst), but Sphinx does
+#   not handle having source files split across the source tree and
+#   build tree.  We could have sym linked everything into the build
+#   tree, but... we didn't.
+
+###########################################################################
+# JMS: to investigate
+
+# handle mpiext affinity and cuda man pages
+###########################################################################
+
+SPHINX_CONFIG      = conf.py
+TEXT_SOURCE_FILES  = $(srcdir)/license/*.txt
+IMAGE_SOURCE_FILES = $(shell find $(srcdir) -name \*.png)
+RST_SOURCE_FILES   = $(shell find $(srcdir) -name \*.rst)
+SPHINX_OPTS       ?= -W --keep-going
+OUTDIR             = _build
+
+CLEANFILES         =
+EXTRA_DIST         = \
+        requirements.txt \
+        $(SPHINX_CONFIG) \
+        $(TEXT_SOURCES)
+
+###########################################################################
+# Only build the docs if we have Sphinx
+if OPAL_BUILD_DOCS
+
+include $(top_srcdir)/Makefile.ompi-rules
+
+# Have to not list these targets in EXTRA_DIST outside of the
+# OPAL_BUILD_DOCS conditional because "make dist" will fail due to
+# these missing targets (and therefore not run the "dist-hook" target
+# in the top-level Makefile, which prints a pretty message about why
+# "make dist" failed).
+#
+# We list the entire directory trees (html and man) to grab all
+# generated files in them.
+EXTRA_DIST += \
+        $(OUTDIR)/html \
+        $(OUTDIR)/man
+
+# List both commands (HTML and man) in a single rule because they
+# really need to be run in serial.  Specifically, if they were two
+# different rules and someone ran "make -j", then both of them could
+# be writing to $(OUTDIR)/doctrees simultaneously, which would be Bad.
+TOP_SENTINEL_FILE = $(OUTDIR)/.built.txt
+$(TOP_SENTINEL_FILE): $(RST_SOURCE_FILES) $(IMAGE_SOURCE_FILES)
+$(TOP_SENTINEL_FILE): $(TEXT_SOURCE_FILES) $(SPHINX_CONFIG)
+$(TOP_SENTINEL_FILE):
+	$(SPHINX_BUILD) -M html "$(srcdir)" "$(OUTDIR)" $(SPHINX_OPTS)
+	$(SPHINX_BUILD) -M man "$(srcdir)" "$(OUTDIR)" $(SPHINX_OPTS)
+	@touch $(OUTDIR)/.built.txt
+
+# A useful rule to invoke manually to ensure that all of the external
+# HTML links we have are valid.  Running this rule requires
+# connectivity to the general internet.
+linkcheck:
+	$(SPHINX_BUILD) -M linkcheck "$(srcdir)" "$(OUTDIR)" $(SPHINX_OPTS)
+
+.PHONY: linkcheck
+
+clean-local:
+	@$(SPHINX_BUILD) -M clean "$(srcdir)" "$(OUTDIR)" $(SPHINX_OPTS)
+
+BUILT_SOURCES = $(TOP_SENTINEL_FILE)
+
+CLEANFILES += $(TOP_SENTINEL_FILE)
+
+endif # OPAL_BUILD_DOCS
+
+###########################################################################
+# Only install the docs if a) we already have them built (e.g., if we're
```

Comment:
Again, this comment is wrong.  The Makefile does what configure says; it has no logic of its own.  This comment belongs in the config macro, not the makefile.